spec/postcss/fixtures/part-nested-multiple-pseudo.pcss

```diff
@@ -0,0 +1,7 @@
+component(frame) {
+  &:hover &, &:focus & {
```

Comment:
I think right now that's not super important because it's an added feature and will likely belong in it's own pull request - this one is just to get feature parity with the Sass version.

I actually posted a comment in the Stylus support issue with a similar idea: https://github.com/geddski/csstyle/issues/1#issuecomment-105562392

Essentially just a mixin wrapper around parent references, you could do `state(hover)` on a component or part or `state(hover, target: some-nested-component-or-part)`